frontend/test/metabase/containers/SaveQuestionModal.unit.spec.js

```diff
@@ -90,12 +90,9 @@ function getDirtyQuestion(originalQuestion) {
     .query()
     .breakout(["field", ORDERS.TOTAL.id, null])
     .question();
-  return question.setCard({
-    ...question.card(),
-    // After a saved question is edited, the ID gets removed
-    // and a user can either overwrite a question or save it as a new one
-    id: undefined,
-  });
+  // After a saved question is edited, the ID gets removed
+  // and a user can either overwrite a question or save it as a new one
+  return question.setId(undefined);
```

Comment:
should it be this? https://github.com/metabase/metabase/blob/bc782f534a72de567b885bb921e4f9fb816c36a9/frontend/src/metabase-lib/Question.ts#L160

Reply:
Right idea, but the wrong method - the name is supposed to be preserved here. We actually want `.markDirty()`, which it now uses.